### ISSUE TITLE
[Request] Delete DXVK state cache feature(s?)

### ISSUE THREAD

**Issue by begin-theadventure** [lutris/lutris#5239]
`Update shader cache`,
1. Graphics Pipeline Library (GPL).
2. `dxvk-cache-tool` is not compatible with versions 2.1+.
3. Read `State cache interactions` (with two notes) in the version [2.0 release](https://github.com/doitsujin/dxvk/releases/tag/v2.0).

**Comment by strycore** [comment]
Please explain better what the goal with this request is here. Make full sentences. I have no idea what you are suggesting here.

**Comment by begin-theadventure** [comment]
1. Lutris has an `Update shader cache` feature, but since version 2.0, DXVK has the Graphics People Library (shortened to GPL), which is mostly used instead of the cache.

2. dxvk-cache-tool, which Lutris uses for updating/merging caches, became incompatible with DXVK version 2.1 and later, and is not maintained.

3. Updating old caches from DXVK 1.10.3 and earlier can cause crashes, as described in the 2.0 release. This is important if Lutris downloads caches generated by those versions.

4. Additionally, I'm guessing that only a few games have caches available, but this option is available for all games/applications from the pop-up menu.
I think it would be better if there was a cache manager in `Preferences` instead that would show available caches for download, etc., but since adding the feature, it has become rather redundant.

**Comment by strycore** [comment]
The update shader cache feature has really only been available for overwatch 2 and even then, we haven't pushed any updates. The update cache feature is only available for games that set a path to the DXVK cache path in the environment variables.

The difference on DXVK 2 is noticeable. I don't think this is needed like it used to be. In my experience, I haven't experienced the type of stuttering we used to experience in a very long time. Granted, I have updated to a machine much more powerful than the 4790k I was using up until recently but even on the Steam Deck, I rarely notice any stuttering.

If the tool we have in place for maintaining state caches is no longer maintained, there's no point for us to continue shipping it.
